pulsar-functions/java-examples/src/main/java/org/apache/pulsar/functions/api/examples/PublishFunctionWithMessageConf.java

```diff
@@ -18,6 +18,8 @@
  */
 package org.apache.pulsar.functions.api.examples;
 
+import org.apache.pulsar.client.api.PulsarClientException;
```

Comment:
can you rename this file to saying typedMessageBuilderPublish

Reply:
sure, cool suggestion, thanks @srkukarni  will fix it